_posts/2020-01-02-announcement-community-forum-events.md

```diff
@@ -10,6 +10,8 @@ The [BIDS Steering Group]({% post_url 2019-12-31-meet-the-bids-steering-group %}
 
 Inspired by [town hall meetings](https://en.wikipedia.org/wiki/Town_hall_meeting) and the OHBM General Assembly and Feedback Forum, these online events are a space for members of the BIDS community to meet the steering group and to discuss BIDS strategy and vision.
 
+UPDATE: We now have recordings of the [first](https://www.youtube.com/watch?v=aBG1kl6iJeE&t=6s) and [second](https://www.youtube.com/watch?v=23OySb5kbtM) town hall meetings available for watching.
```

Comment:
suggestion to preserve consistency with Community Forum (with the C and F capped as we have so far referred to it)

I also wonder if it may be better to move this to the top rather than embedded?

```suggestion
UPDATE: We have shared recordings of the [first](https://www.youtube.com/watch?v=aBG1kl6iJeE&t=6s) and [second](https://www.youtube.com/watch?v=23OySb5kbtM) Community Forum meetings.
```